src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Internal/Controllers/InternalConnectionsController.cs

```diff
@@ -0,0 +1,217 @@
+using System.Net.Mime;
+using Altinn.AccessManagement.Api.Internal.Models;
+using Altinn.AccessManagement.Core.Constants;
+using Altinn.AccessManagement.Core.Errors;
+using Altinn.AccessManagement.Core.Models;
+using Altinn.AccessManagement.Internal.Services;
+using Altinn.AccessMgmt.Core.Models;
+using Altinn.AccessMgmt.Persistence.Core.Models;
+using Altinn.AccessMgmt.Persistence.Data;
+using Altinn.AccessMgmt.Persistence.Services;
+using Altinn.AccessMgmt.Persistence.Services.Models;
+using Altinn.Authorization.ProblemDetails;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.FeatureManagement.Mvc;
+
+namespace Altinn.AccessManagement.Api.Internal.Controllers;
+
+/// <summary>
+/// Controller for en user api operations for connections
+/// </summary>
+[ApiController]
+[Route("accessmanagement/api/v1/internal/connections")]
```

Comment:
Hm.. 🤔 
Innser at vi kanskje burde være mer spesifikke på path og funksjonaliteten i disse endepunktene.

Disse endepunktene skal _bare_ kunne brukes for å hente, legge til eller fjerne tilganger for systembrukere for en gitt from-organisasjon. Så andre connections skal ikke kunne endres her.

Så vi burde kanskje være mer spesifikke, noe ala:
`accessmanagement/api/v1/internal/systemuser-connections`

Reply:
hmm, usikker. ikke alt for faan av å bryte pattern :p